quarkchain/cluster/tests/test_cluster.py

```diff
@@ -2430,3 +2451,91 @@ def mock_get_root_chain_stakes(recipient, _):
             # fail again, because quota used up
             with self.assertRaises(ValueError):
                 add_root_block(staker_addr, sign=True)
+
+    def test_grpc_slave(self):
+        id1 = Identity.create_random_identity()
+        acc1 = Address.create_from_identity(id1, full_shard_key=0)
+
+        with ClusterContext(1, acc1, connect_grpc=True) as clusters:
+            block_header_list = [clusters[0].get_shard_state(2 | 0).header_tip]
```

Comment:
we don't really need this setup. a simple root block should be enough. can remove line from 2460 to 2481 using empty minor block header list